deployments/liqo/templates/webhooks/job-patch/job-create-secret.yaml

```
@@ -37,3 +37,15 @@ spec:
       serviceAccountName: {{ include "liqo.prefixedName" $rbacConfig }}
       securityContext:
         {{- include "liqo.podSecurityContext" . | nindent 8 }}
+      {{- if ((.Values.common).nodeSelector) }}
```

```suggestion
      {{- if (.Values.common.nodeSelector) }}
```

Are those () required? (the same for the other occurrences)

---

Yes. Just for backward compatibility. If no ‘common’ field doesn’t exist it should not break installation.